tower/src/spawn_ready/service.rs

```diff
@@ -1,4 +1,4 @@
-use super::future::ResponseFuture;
+use super::{future::ResponseFuture, SpawnReadyLayer};
```

Comment:
clippy says this import is unused, because the `SpawnReady::layer` function references it from `super`:
https://github.com/tower-rs/tower/blob/49d77909515dbc4f5962f4cdfe5eaac7a89836b2/tower/src/spawn_ready/service.rs#L35-L37

we should remove this (or remove the `super` in `Layer`, I don't really care).
```suggestion
use super::future::ResponseFuture;
```